front/src/modules/trainschedule/components/DriverTrainScheduleV2/DriverTrainScheduleStopV2.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useMemo } from 'react';
+
+import cx from 'classnames';
+
+import type { SimulationResponseSuccess } from 'applications/operationalStudies/types';
+import type { PositionSpeedTime } from 'reducers/osrdsimulation/types';
+import { mmToM } from 'utils/physics';
+
+import type { OperationalPointWithTimeAndSpeed } from './types';
+import { getActualSpeed, getActualVmax, getAverageSpeed, getTime, isEco } from './utils';
+
+type DriverTrainScheduleStopV2Props = {
+  operationalPoint: OperationalPointWithTimeAndSpeed;
+  operationalPoints: OperationalPointWithTimeAndSpeed[];
+  idx: number;
+  simulatedTrain: SimulationResponseSuccess;
+};
+
+const DriverTrainScheduleStopV2 = ({
+  operationalPoint,
+  operationalPoints,
+  idx,
+  simulatedTrain,
+}: DriverTrainScheduleStopV2Props) => {
+  const trainRegime = isEco(simulatedTrain) ? simulatedTrain.final_output : simulatedTrain.base;
+
+  const formattedTrainRegimeReports: PositionSpeedTime[] = useMemo(
+    () =>
+      trainRegime.positions.map((position, index) => ({
+        position: mmToM(position),
+        speed: trainRegime.speeds[index],
+        time: trainRegime.times[index],
+      })),
+    [trainRegime]
+  );
+
+  const formattedMrsp = useMemo(
+    () =>
+      simulatedTrain.mrsp.positions.map((_position, index) => ({
+        position: _position,
+        speed: simulatedTrain.mrsp.speeds[index],
+      })),
+    [simulatedTrain.mrsp]
+  );
```

Comment:
Move this code to the parent component
```suggestion
  const formattedTrainRegimeReports: PositionSpeedTime[] = useMemo(
    () =>
      trainRegime.positions.map((position, index) => ({
        position: mmToM(position),
        speed: trainRegime.speeds[index],
        time: trainRegime.times[index],
      })),
    [trainRegime]
  );

  const formattedMrsp = useMemo(
    () =>
      simulatedTrain.mrsp.positions.map((_position, index) => ({
        position: _position,
        speed: simulatedTrain.mrsp.speeds[index],
      })),
    [simulatedTrain.mrsp]
  );
```

Reply:
What does it change ? This would mean more props to pass and the parent component won't benefit from them.

Reply:
Ok I get it, then it won't be compute for each operational point, good catch !